NetKVM/Common/ParaNdis_Common.cpp

```diff
@@ -785,11 +785,6 @@ NDIS_STATUS ParaNdis_InitializeContext(
 
         InitializeLinkPropertiesConfig(pContext);
 
-#if !defined(_ARM64_)
```

Comment:
Sorry, but the second commit still removes the needed line.

You should keep this:
```
pContext->bControlQueueSupported = AckFeature(pContext, VIRTIO_NET_F_CTRL_VQ);
```

You can checkout all PR and see also the final result: https://docs.github.com/en/enterprise/2.15/user/articles/checking-out-pull-requests-locally

In any case, line 787 after "InitializeLinkPropertiesConfig(pContext);" line should be:
pContext->bControlQueueSupported = AckFeature(pContext, VIRTIO_NET_F_CTRL_VQ);

Reply:
This does not make sense to me. When I look at the combined changes in "files" I see no such code removal:

https://github.com/virtio-win/kvm-guest-drivers-windows/pull/479/files

![image](https://user-images.githubusercontent.com/40605312/87233232-8a2f3f00-c37a-11ea-9a5b-a34529496da9.png)

Reply:
@YanVugenfirer , I see what is going on. The line was also missing even before mediouni-m's commit.

Reply:
Re-added.

Reply:
I also squashed all the commits, so it is now pretty obvious that my change was not removing the line. It was just not adding it.

Reply:
I am rebuilding the driver and testing it with control queues now.

Reply:
Tested - new driver with line added works.